plugins/reporters/evaluated-model/src/funTest/kotlin/EvaluatedModelReporterFunTest.kt

```diff
@@ -19,57 +19,52 @@
 
 package org.ossreviewtoolkit.plugins.reporters.evaluatedmodel
 
-import io.kotest.core.TestConfiguration
 import io.kotest.core.spec.style.WordSpec
 import io.kotest.engine.spec.tempdir
 import io.kotest.matchers.shouldBe
 
-import org.ossreviewtoolkit.model.FileFormat
 import org.ossreviewtoolkit.model.OrtResult
-import org.ossreviewtoolkit.plugins.api.PluginConfig
 import org.ossreviewtoolkit.reporter.ReporterInput
 import org.ossreviewtoolkit.utils.common.normalizeLineBreaks
 import org.ossreviewtoolkit.utils.test.getAssetAsString
 import org.ossreviewtoolkit.utils.test.readOrtResult
 
 class EvaluatedModelReporterFunTest : WordSpec({
+    fun EvaluatedModelReporter.generateReport(ortResult: OrtResult) {
```

Comment:
Nit: Could this stay to be a private top-level function at the bottom of the class to reduce the diff and nesting a bit?

Reply:
On the top-level it would not have access to the `tempdir()` function from Kotest.